jvector-examples/src/main/java/io/github/jbellis/jvector/example/util/DownloadHelper.java

```diff
@@ -78,10 +73,13 @@ public static void maybeDownloadFvecs(List<String> files) {
                                 .build();
 
                 // 3 retries
+                FileDownload downloadFile;
```

Comment:
these aren't used outside the loop body so why move the declaration outside?

Reply:
I was trying to add something and left it outside. I'll put it back.